project-setup/templates/README-template.md

```diff
@@ -0,0 +1,73 @@
+# Insert the name of your project here
+
+## Mission
+
+This should contain a brief (3-5 sentences) description of the mission of your
+project. The goal is to state what you are planning to work on and help external
+contributors understand roughly which types of features will likely be welcome
+for this project.
+
+See also [mission statement
+chapter](https://producingoss.com/en/producingoss.html#mission-statement) in
+Producing Open Source Software.
+
```

Comment:
This could include properties and qualities that want to be achieved in this project. Some of those qualities can be very specific and may be changed over the lifetime of the project. They help identify what the project wants to achieve and how what tradeoffs are chosen. For example "in order to strive for simplicity and performance we may not implement feature request that improve convenience".

Reply:
I believe this goes a bit too far for a mission statement. However I believe this is valuable information to include. I have moved it to the contributing section as it is information that contributors will need to make in informed decision about whether their change will be accepted or not.

Writing this text down, I've come to think that this may well be interesting to users who never thought about contributing as well - it could well help decide whether or not to adopt the software.

Likely the template needs some more user focused information.